src/coverage/configureProject.ts

```diff
@@ -69,7 +78,9 @@ export async function configureProjectWithGcov(workspacePath: Uri) {
     appTraceGcovEnable === "y";
 
   if (isGcovEnabled) {
-    return window.showInformationMessage("Code coverage is already enabled in sdkconfig");
+    return window.showInformationMessage(
```

Comment:
Not sure why this file is modified in this PR. Probably a rebase thing ?

These changes are ok but unrelated to PR focus.

Reply:
As I've mentioned in another comment, you are right, I've reverted the changes, so they are aligned with the "master" branch.